config/routes.rb

```diff
@@ -4,9 +4,12 @@
     patch :instructions, to: "assistants/instructions#update"
   end
 
-  resources :conversations do
-    resources :messages, shallow: true
+  shallow do
+    resources :conversations do
+      resources :messages
+    end
```

Comment:
The way I saw this explained in the rails guide, you either put shallow: true on the inner route or the outer route. I'm not sure what the `shallow do` block does:
https://guides.rubyonrails.org/routing.html#shallow-nesting

Reply:
The docs on shallow nesting provide a lot of options for expression. Near the bottom is the `shallow` method, which they say does the same thing as putting the shallow flag on individual routes:

<img width="679" alt="image" src="https://github.com/hostedgpt/hostedgpt/assets/208647/493c8100-2fde-401b-a5f9-dc7e51a63bb1">

I selected this one because it felt like it would be easier to add more resources under the pattern than adding `shallow: true` to each. Perhaps that's a premature move, since our data model is likely to stay so lean.